var/spack/repos/builtin/packages/abyss/package.py

```diff
@@ -33,9 +33,12 @@ class Abyss(AutotoolsPackage):
     homepage = "http://www.bcgsc.ca/platform/bioinfo/software/abyss"
     url      = "https://github.com/bcgsc/abyss/archive/2.0.2.tar.gz"
 
-    version('2.0.2', '1623f55ad7f4586e80f6e74b1f27c798')
+    version('2.0.2', 'bb3f8cebf121312bf81789d963b4ecc5')
```

Comment:
Just curious: why the change in the hash?

Reply:
When I uninstalled and tried to re-install, spack returned an error saying that the hash was different than expected

Reply:
Maybe another issue from #5411 ...